doc/api/cli.md

```diff
@@ -51,6 +51,53 @@ entry point, the `node` command will accept as input only files with `.js`,
 [`--experimental-wasm-modules`][] is enabled; and with no extension when
 [`--experimental-default-type=module`][] is passed.
 
+## Sub-commands
+
+### `node run <command> [<arguments>]`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.1 - Active development
+
+This runs a specified command from a package.json's `"scripts"` object.
+If no `"command"` is provided, it will list the available scripts.
+
+`node run` prepends `./node_modules/.bin`, relative to the current
+working directory, to the `PATH` in order to execute the binaries from
+dependencies.
+
+For example, the following command will run the `test` script of
+the `package.json` in the current folder:
+
+```console
+$ node run test
+```
+
+You can also pass arguments to the command. Any argument after the script
+name will be appended to the script:
+
+```console
+$ node run test --verbose
+```
+
+#### Intentional limitations
+
+`node run` is not meant to match the behaviors of `npm run` or of the `run`
+commands of other package managers. The Node.js implementation is intentionally
+more limited, in order to focus on top performance for the most common use
+cases.
+Some features of other `run` implementations that are intentionally excluded
+are:
+
+* Searching for `package.json` files outside the current folder.
+* Prepending the `.bin` or `node_modules/.bin` paths of folders outside the
+  current folder.
+* Running `pre` or `post` scripts in addition to the specified script.
+* Defining package manager-specific environment variables.
+* Setting `process.title`.
```

Comment:
What does this mean to users though? Either we find it unnecessary for a feature coming from Node.js core, then we can discard this line, or we should explain the security implication of it.

Reply:
I'm fine with removing this line. cc @GeoffreyBooth